resources/views/components/transaction/page/token.blade.php

```diff
@@ -0,0 +1,41 @@
+@props(['transaction'])
+
+<x-general.page-section.container :title="trans('pages.transaction.tokens_transferred')">
+    <x-transaction.page.section-detail.row
+        :title="trans('pages.transaction.header.to')"
+        :transaction="$transaction"
+    >
+        <x-transaction.page.section-detail.address
+            :address="\App\Services\ContractPayload::decodeAddress($transaction->methodArguments()[0])"
+            class="inline-block"
+        />
+    </x-transaction.page.section-detail.row>
+
+    <x-transaction.page.section-detail.row
+        :title="trans('pages.transaction.header.amount')"
+        :value="ExplorerNumberFormatter::weiToArk(\App\Services\ContractPayload::decodeUnsignedInt($transaction->methodArguments()[1]))"
```

Comment:
kinda ugly having all of this in the template 🤔 also hard to follow with the indexes. Maybe change them to an enum so it gives some context as to what that position includes?